docs/guides/getting-started/prerequisites.md

```diff
@@ -125,6 +125,20 @@ sudo dnf install webkit2gtk3-devel.x86_64 \
 sudo dnf group install "C Development Tools and Libraries"
 ```
 
+  </TabItem>
+  <TabItem value="opensuse" label="openSUSE">
+
+```sh
+sudo zypper up
+sudo zypper in patterns-devel-base-devel_basis \
+    webkit2gtk3-soup2-devel \
+    libopenssl-devel \
+    curl \
+    wget \
+    libappindicator3-1 \
+    librsvg-devel
```

Comment:
What do you think about showing the default way for the pattern installation? This would align it with the fedora instructions, so it's not too alien x)
```suggestion
sudo zypper up
sudo zypper in webkit2gtk3-soup2-devel \
    libopenssl-devel \
    curl \
    wget \
    libappindicator3-1 \
    librsvg-devel
sudo zypper in -t pattern devel_basis
```

Reply:
Wow... Honestly, I've been using openSUSE for 3 years and never knew about the `-t pattern` syntax. I guess you learn something new every day! :smile: